src/conditional-flow/multiple-conditions.js

```diff
@@ -2,7 +2,7 @@
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
 function isInRange (num, lower, upper) {
-
+return (lower<=num&&num<=upper);            
```

Comment:
Just a note - the brackets aren't really needed here, but if you find it easier to read like this then that's no problem :)